serial-it.sh

```diff
@@ -90,15 +103,12 @@ case $BOARD in
         serialdev=ttyAMA0
         baudrate=9600
         ;;
-    raspberrypi3)
```

Comment:
Let's stay consistent with slugs in device types. https://jenkins.dev.resin.io/view/Yocto/job/yocto-raspberrypi3/lastSuccessfulBuild/artifact/deploy/device-type.json/*view*/